src/components/navigation/Navigations.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useContext } from "react";
+import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
+import Icon from "@expo/vector-icons/AntDesign";
+import { ServiceContext } from "../../store/ServiceContext";
+
+import Settings from "../../screens/Settings";
+import globalStyles from "../../css/global";
+
+
+const Navigations = () => {
+    const [modalVisible, setModalVisible] = useState(false);
+    const { serviceSelected } = useContext(ServiceContext);
+    const mode = serviceSelected;
+
+    const showSettings = () => {
+        setModalVisible(true);
+    };
+
+    const hideSettings = () => {
+        setModalVisible(false);
+    };
+
+    return (
+        <View style={styles.container}>
+            <TouchableOpacity
+                style={[styles.button, globalStyles.button[mode]]}
+                onPress={showSettings}
+            >
+                <Text style={styles.text}>settings<Icon name="setting" size={15} color="white" /></Text>
```

Comment:
It should be Icon then text. make sure to add margins as well.
![image](https://github.com/kebluis/eod-report-pomodoro-client/assets/75247295/3cc85a12-7630-442f-a20d-07e96ce5e830)